src/plugins/console/server/lib/proxy_config_collection.ts

```diff
@@ -29,23 +29,23 @@
  */
 
 import { defaultsDeep } from 'lodash';
-import { parse as parseUrl } from 'url';
```

Comment:
Not for this PR, but we should remove this dependency from our codebase

Reply:
Created issue to track: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1561